### PR TITLE
WIP: scaffold for Circuit Breaker mission for all 3 runtimes

### PR DESCRIPTION
**Pull request by inoxx03**
* Scaffold added to `master.adoc` files
* guide name variables defined and added to `document-attributes.adoc` file
* Drafts of individual guide sections to be added in subsequent commits

### CODE REVIEW

**Comment by edewit** [comment]
@inoxx03 can we have a circuit-breaker-mission-intro-paragraph.adoc that contains the first paragraph of the into so that we can use that on the front-end?

File: docs/topics/circuit-breaker-mission-intro-paragraph.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+The _Circuit Breaker_ Mission demonstrates a generic pattern for reporting the failure of a service and then limiting access to the failed service until it becomes available to handle requests. This helps prevent cascading failure in other services that depend on the failed services for functionality.
```
Comment (inoxx03) [inline docs/topics/circuit-breaker-mission-intro-paragraph.adoc]: @edewit WDYT? Would this be OK as the intro paragraph. Please let me know if you want me to shorten/expand/correct it.

**Comment by inoxx03** [comment]
@edewit I've added  a draft of the introductory paragraph and also some text for the intro section. Please let me know what you think.

File: docs/topics/circuit-breaker-mission-intro-paragraph.adoc (hunk 1 above)
Comment (gastaldi) [inline docs/topics/circuit-breaker-mission-intro-paragraph.adoc]: LGTM

Comment (edewit) [inline docs/topics/circuit-breaker-mission-intro-paragraph.adoc]: 👍

**Comment by Ladicek** [comment]
Looks good to me.

**Pull request merged**